ngi_reports/stockholm/project_summary.py

```diff
@@ -260,12 +365,14 @@ def get_accredit_info(self,keys):
             try:
                 accredit = self.proj_details['accredited_({})'.format(k)]
                 if accredit in ['Yes','No']:
-                    accredit_info[k] = "{} Validated under ISO accreditation 17025:2005".format(["[cross]","[tick]"][accredit == "Yes"])
+                    accredit_info[k] = "{} under ISO accreditation 17025:2005".format(["[cross] Not validated","[tick] Validated"][accredit == "Yes"])
                 else:
-                    accredit_info[k] = accredit
+                    accredit_info[k] = accredit.replace("N/A","Not Applicable")
             except KeyError:
-                self.LOG.error("Could not find accreditation info for step {} for project {}".format(k,self.project_name))
-                accredit_info[k] = ''
+                if k in ['library_preparation','data_analysis'] and self.proj.get('application') == "Finished library":
+                    accredit_info[k] = "Not Applicable"
+                else:
```

Comment:
this is check an exception if `PCO` forgets to choose something and if project is a `finished library` we don't have to bring it up and fix it before re-generating the report.